pkg/executor/api.go

```diff
@@ -244,6 +244,20 @@ func (executor *Executor) unTapService(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+// dumpFunctionService => dump function service for pool cache
+func (executor *Executor) dumpFunctionService(w http.ResponseWriter, r *http.Request) {
+	// currently we are considering dumping function only for pool manager
+	et := executor.executorTypes[fv1.ExecutorTypePoolmgr]
+	if err := et.DumpFnSvcCache(r.Context()); err != nil {
```

Comment:
Please rename all methods to DumpDebugInfo as we want to keep these generic for future purpose.